doc/api/cli.md

```diff
@@ -331,6 +331,53 @@ reference. Code may break under this flag.
 `--require` runs prior to freezing intrinsics in order to allow polyfills to
 be added.
 
+### `--heapsnapshot-near-heap-limit=max_count`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Writes a V8 heap snapshot to disk when the V8 heap usage is approaching the
+heap limit. `count` should be either -1 (in which case no snapshots would be
+written), or a positive integer (in which case Node.js will write no more than
+`max_count` snapshots to disk).
```

Comment:
Why was `-1` chosen over `0`?

Reply:
The intention was actually to only allow positive integers but we also allow -1 because we need a "undefined" value in the the option parser and in this case I set it to -1 - so this is just an exception. Maybe we should just leave -1 undocumented and ask the author to specify a positive integer?

Reply:
¯\\_(ツ)\_/¯ I would naively assume `0` would be off (i.e. zero/no snapshots written).

Reply:
OK, I set the "undefined" value to 0, it becomes a noop when the value is 0

Reply:
Thanks.